### DISCUSSION TITLE
Support for other API providers (Mistral)

### DISCUSSION THREAD

**Discussion by LEBaz2211** [logancyang/obsidian-copilot#242] [Ideas]
Hey there!

Mistral AI just launched their own API endpoints and it would be great if we could be able to choose the base url of the target endpoint (as it is the only thing we have to change with the model names of course, they use the same API as OpenAI) so that we could use these new gorgeous gorgeous models 😁

Thanks for the incredible work so far ! Your plugin is just great!
Have a nice day,
Basil Mannaerts

**Comment by logancyang** [comment]
Thanks for using the plugin! This task is tracked here https://github.com/logancyang/obsidian-copilot/issues/238

**Reply by LEBaz2211** [reply]
Thank you ! And sorry I didn't see that
